### PR TITLE
White label scripts

### PR DESCRIPTION
**Pull request by CalebPena**
What (if any) features are you implementing?
- Update `add_config` script to use a white label.
- Update `validate` script to use a white label.
- Update `health_check` script to use a white label.
- Update `bulk_import` and `bulk_export` scripts to use a white label.
- Update hubspot scripts to use a white label.

What (if anything) did you refactor?
- Remove old scripts.

### CODE REVIEW

File: configuration/management/commands/add_config.py
Comment (ivonne-hernandez) [inline configuration/management/commands/add_config.py:1]: If we need to change a state's config values these will now need to be changed in the `configuration/white_labels/co.py` file, but do we still run the same `python3 manage.py add_config` script to implement the changes?

Comment (CalebPena) [reply configuration/management/commands/add_config.py:1]: Run `python3 manage.py add_config --all` to update the config for all of the white labels, or `python3 manage.py add_config co` to update the config for just Colorado.

File: configuration/white_labels/__init__.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from ._default import DefaultConfiguarationData
+from .base import ConfigurationData
+from .co import CoConfiguarationData
+from .nc import NcConfiguarationData
+
+
+white_label_config: dict[str, ConfigurationData] = {
+    "co": CoConfiguarationData,
+    "nc": NcConfiguarationData,
+    "_default": DefaultConfiguarationData,
```
Comment (ivonne-hernandez) [inline configuration/white_labels/__init__.py:10]: What do you think about fixing the `Configuration` spelling on all of these except lines 2 & 7?

File: configuration/white_labels/_default.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+from .base import ConfigurationData
+from screener.models import WhiteLabel
+
+
+class DefaultConfiguarationData(ConfigurationData):
```
Comment (ivonne-hernandez) [inline configuration/white_labels/_default.py:5]: What do you think about fixing the `Configuration` spelling?

File: configuration/white_labels/base.py [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+    @classmethod
+    def get_white_label(self) -> WhiteLabel:
+        raise NotImplemented()
+
+    public_charge_rule = {"link": ""}
+
+    more_help_options = {
+        "moreHelpOptions": [
+            {
+                "name": {"_default_message": "", "_label": ""},
```
Comment (ivonne-hernandez) [inline configuration/white_labels/base.py:16]: Why do we want these white label base values to be empty strings?

Comment (CalebPena) [reply configuration/white_labels/base.py:16]: The base is just to give an idea of what the structure should look like. It shouldn't be imported as part of the config.

**Review by ivonne-hernandez — APPROVED** [review]

**Pull request merged**